src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+@keyframes minutes {
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes seconds {
```

Comment:
You have 2 absolutely the same keyframes, you can use one of them and nothing will change